sile-dev-1.rockspec

```diff
@@ -9,11 +9,10 @@ source = {
 }
 dependencies = {
   "lua >= 5.1",
-  "bit32",
```

Comment:
compat53 does not bundle bit32, because bit32 is deprecated in Lua 5.3. The code lives in the same repository, but there are still separate rockspecs for both.

Reply:
Thanks. I figured that out after I tried this, but didn't push the change because I didn't actually figure out how to get it loaded.

Lua 5.3 only included `bit32` if/when compiled with 5.2 compatibility (which was the default). 5.4 doesn't seem to have it at all by default ... which leads me to wonder ... it seems this _IS_ still needed in order to write code that works across Lua versions.